ut/profile/hdmiCECExtendedEnumsNotSupported.yaml

```diff
@@ -0,0 +1,3 @@
+hdmicec:
```

Comment:
this is in the wrong place.

Look at your directory structure when you clone this repo.

header(rdk-halif-hdmi_cec)->ut(rdk-halif-test-hdmi_cec)->ut_core

So this directory isn't `ut/profile` from this git repo point of view it's `profile`